src/lib/extend/item/getSKU.ts

```diff
@@ -322,3 +327,95 @@ function getTarget(item: EconItem, schema: SchemaManager.Schema): number | null
+
+    const crates: { [type: string]: { [name: string]: number } } = {
+        is5022: {
+            'Mann Co. Supply Crate Series #1': 1,
+            'Mann Co. Supply Crate Series #3': 3,
+            'Mann Co. Supply Crate Series #7': 7,
+            'Mann Co. Supply Crate Series #12': 12,
+            'Mann Co. Supply Crate Series #13': 13,
+            'Mann Co. Supply Crate Series #18': 18,
+            'Mann Co. Supply Crate Series #19': 19,
+            'Mann Co. Supply Crate Series #23': 23,
+            'Mann Co. Supply Crate Series #26': 26,
+            'Mann Co. Supply Crate Series #31': 31,
+            'Mann Co. Supply Crate Series #34': 34,
+            'Mann Co. Supply Crate Series #39': 39,
+            'Mann Co. Supply Crate Series #43': 43,
+            'Mann Co. Supply Crate Series #47': 47,
+            'Mann Co. Supply Crate Series #54': 54,
+            'Mann Co. Supply Crate Series #57': 57,
+            'Mann Co. Supply Crate Series #75': 75
+        },
+        is5041: {
+            'Mann Co. Supply Crate Series #2': 2,
+            'Mann Co. Supply Crate Series #4': 4,
+            'Mann Co. Supply Crate Series #8': 8,
+            'Mann Co. Supply Crate Series #11': 11,
+            'Mann Co. Supply Crate Series #14': 14,
+            'Mann Co. Supply Crate Series #17': 17,
+            'Mann Co. Supply Crate Series #20': 20,
+            'Mann Co. Supply Crate Series #24': 24,
+            'Mann Co. Supply Crate Series #27': 27,
+            'Mann Co. Supply Crate Series #32': 32,
+            'Mann Co. Supply Crate Series #37': 37,
+            'Mann Co. Supply Crate Series #42': 42,
+            'Mann Co. Supply Crate Series #44': 44,
+            'Mann Co. Supply Crate Series #49': 49,
+            'Mann Co. Supply Crate Series #56': 56,
+            'Mann Co. Supply Crate Series #71': 71,
+            'Mann Co. Supply Crate Series #76': 76
+        },
+        is5045: {
+            'Mann Co. Supply Crate Series #5': 5,
+            'Mann Co. Supply Crate Series #9': 9,
+            'Mann Co. Supply Crate Series #10': 10,
+            'Mann Co. Supply Crate Series #15': 15,
+            'Mann Co. Supply Crate Series #16': 16,
+            'Mann Co. Supply Crate Series #21': 21,
+            'Mann Co. Supply Crate #Series 25': 25,
+            'Mann Co. Supply Crate #Series 28': 28,
+            'Mann Co. Supply Crate Series #29': 29,
+            'Mann Co. Supply Crate Series #33': 33,
+            'Mann Co. Supply Crate Series #38': 38,
+            'Mann Co. Supply Crate Series #41': 41,
+            'Mann Co. Supply Crate Series #45': 45,
+            'Mann Co. Supply Crate Series #55': 55,
+            'Mann Co. Supply Crate Series #59': 59,
+            'Mann Co. Supply Crate Series #77': 77
+        },
+        is5068: {
+            'Salvaged Mann Co. Supply Crate #30': 30,
+            'Salvaged Mann Co. Supply Crate #40': 40,
+            'Salvaged Mann Co. Supply Crate #50': 50
+        }
```

Comment:
I don't really like the hardcoding, could something be done about it?

Reply:
I'll leave it to you or others. Schema doesn't have this.